crane-cloud/frontend
# fix: make app summary details [appUrl & port] dynamic

# Description

- The goal of this PR is to make the values of the `appUrl` and `port` dynamic in the event of an update being made on an application since previously those values were static.

## Type of change


## Trello Ticket ID

https://trello.com/c/eWQKHZSG

## How Can This Been Tested?

- Checkout this branch locally and visit you deployed application's dashboard, and make you can attempt to make an update on the port values and appUrl by for example adding a custom domain. Technically those new values should reflect on that app metrics dashboard.

## Checklist:


# Screenshots

https://github.com/crane-cloud/frontend/assets/63339234/81140398-2d9d-41d0-b89c-7a6d190dd0b6
